vnext/ReactUWP/Modules/AppThemeModuleUwp.cpp

```diff
@@ -50,12 +63,46 @@ const std::string AppTheme::getCurrentTheme()
   return m_currentTheme == winrt::ApplicationTheme::Light ? AppTheme::light : AppTheme::dark;
 }
 
+bool AppTheme::getIsHighContrast()
+{
+  return m_accessibilitySettings.HighContrast();;
+}
+
+// Returns the RBG values for the 8 relevant High Contrast elements.
+folly::dynamic AppTheme::getHighContrastColors() {
+  winrt::Windows::UI::Color ButtonFaceColor = m_uiSettings.UIElementColor(winrt::UIElementType::ButtonFace);
+  winrt::Windows::UI::Color ButtonTextColor = m_uiSettings.UIElementColor(winrt::UIElementType::ButtonText);
+  winrt::Windows::UI::Color GrayTextColor = m_uiSettings.UIElementColor(winrt::UIElementType::GrayText);
+  winrt::Windows::UI::Color HighlightColor = m_uiSettings.UIElementColor(winrt::UIElementType::Highlight);
+  winrt::Windows::UI::Color HighlightTextColor = m_uiSettings.UIElementColor(winrt::UIElementType::HighlightText);
+  winrt::Windows::UI::Color HotlightColor = m_uiSettings.UIElementColor(winrt::UIElementType::Hotlight);
+  winrt::Windows::UI::Color WindowColor = m_uiSettings.UIElementColor(winrt::UIElementType::Window);
+  winrt::Windows::UI::Color WindowTextColor = m_uiSettings.UIElementColor(winrt::UIElementType::WindowText);
+  
+  folly::dynamic rbgValues = folly::dynamic::object("ButtonFaceColor", formatRGB(ButtonFaceColor))
+                                                  ("ButtonTextColor", formatRGB(ButtonTextColor))
+                                                  ("GrayTextColor", formatRGB(GrayTextColor))
+                                                  ("HighlightColor", formatRGB(HighlightColor))
+                                                  ("HighlightTextColor", formatRGB(HighlightTextColor))
+                                                  ("HotlightColor", formatRGB(HotlightColor))
+                                                  ("WindowColor", formatRGB(WindowColor))
+                                                  ("WindowTextColor", formatRGB(WindowTextColor));
+  return rbgValues;
+}
+
+std::string AppTheme::formatRGB(winrt::Windows::UI::Color ElementColor) {
+  std::stringstream RGBString;
+  RGBString << "#" << std::setfill('0') << std::setw(2) << std::hex << (int)ElementColor.R 
+            << std::setfill('0') << std::setw(2) << std::hex << (int)ElementColor.G 
+            << std::setfill('0') << std::setw(2) << std::hex << (int)ElementColor.B;
+  return RGBString.str();
+}
```

Comment:
Our version of this code does this:
 char colorChars[10];
 sprintf_s(colorChars, "#%02x%02x%02x%02x", color.R, color.G, color.B, color.A);
though whichever you find more readable is probably fine.